### PR TITLE
Feature/pp9 8712

### PR DESCRIPTION
**Pull request by tapmapp**

### CODE REVIEW

File: docs/picturepark-sdk-v1-angular/pp-content-aggregation-list.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# pp-content-aggregation-list
+
+The Content Aggregation List
+It extends from the AggregationListComponent in order to inherit common properties and functionalities of the
+
+```
+<pp-content-aggregation-list [browser]="self" [entityName]="'contents'"></pp-content-aggregation-list>
```
Comment (neotek) [inline docs/picturepark-sdk-v1-angular/pp-content-aggregation-list.md]: Remove

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/browser-toolbar/browser-toolbar.component.scss [hunk 2]
```diff
@@ -25,7 +25,7 @@
         }
 
         .order-by {
-
+            margin: 0px 10px 0px 0px;
```
Comment (neotek) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/browser-toolbar/browser-toolbar.component.scss]: Move and override in the app.

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/list-browser/list-browser.component.scss [hunk 3]
```diff
@@ -37,6 +37,16 @@
     .mat-row:hover {
       background-color: lightgray;
     }
+
+    &::-webkit-scrollbar {
+      width: 10px;
+    }
+    
+    &::-webkit-scrollbar-thumb {
+      background-color: darkgrey;
```
Comment (neotek) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/list-browser/list-browser.component.scss]: Add colors to theme.

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/panel/components/about-pp/about-pp.component.html [hunk 4]
```diff
@@ -0,0 +1,28 @@
+<div class="panel-container">
```
Comment (neotek) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/panel/components/about-pp/about-pp.component.html]: Move to app.

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/panel/components/help-support-resources/help-support-resources.component.html [hunk 5]
```diff
@@ -0,0 +1,13 @@
+<div class="panel-container">
+  <div class="panel-header">
+    <h3>{{ title }}</h3>
+  </div>
+  <div class="panel-content">
+    <p>
+      Please always consult the Picturepark Documentation prior to filing a support query. In case your Picturepark SLA or support is provided by a partner, then please contact your partner’s support organization before you contact Picturepark Support directly. 
+    </p>
+    <p>
+      <a href="">Picturepark Documentation & Support</a>
```
Comment (neotek) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/panel/components/help-support-resources/help-support-resources.component.html]: Fix the link

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/panel/components/system-information/system-information.component.html [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+  <div class="panel-header">
+    <h3>{{ title }}</h3>
+  </div>
+  <div class="panel-content">
+    <p>
+      When referring your Picturepark instance for support or any other queries with us or our partners, please ensure to always include the following information together with a precise description: 
+    </p>
+    
+    <ul>
+      <li><span>Cloud Service Version:</span> 10.3.0+9609bea765</li>
```
Comment (neotek) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/panel/components/system-information/system-information.component.html]: Get from backend.

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/panel/components/third-party-licenses-credits/third-party-licenses-credits.component.html [hunk 7]
```diff
@@ -0,0 +1,185 @@
+<div class="panel-container">
```
Comment (neotek) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/panel/components/third-party-licenses-credits/third-party-licenses-credits.component.html]: Remove

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/panel/panel.module.ts [hunk 8]
```diff
@@ -10,14 +10,26 @@ import { ItemsPanelComponent } from './components/items-panel/items-panel.compon
 import { OwnerPanelComponent } from './components/owner-panel/owner-panel.component';
 import { ItemPanelPreviewComponent } from './components/items-panel/components/item-panel-preview/item-panel-preview.component';
 import { MailRecipientsPanelComponent } from './components/mail-recipients-panel/mail-recipients-panel.component';
+import { HelpSupportResourcesComponent } from './components/help-support-resources/help-support-resources.component';
```
Comment (neotek) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/panel/panel.module.ts]: Remove and add to app.

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/components/aggregation-list/aggregation-list.component.scss [hunk 9]
```diff
@@ -32,6 +32,19 @@
       justify-content: space-between;
       font-size: 14px;
       background-color: #eeeeee;
+      padding: 0 20px;
     }
   }
+
+  mat-toolbar {
```
Comment (neotek) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/components/aggregation-list/aggregation-list.component.scss]: Move to app.

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/components/aggregation/aggregation.component.scss [hunk 10]
```diff
@@ -41,9 +54,31 @@
...
+        &.mat-pseudo-checkbox-checked {
+          border: 2px solid #5cbebd;
+        }
       }
     }
+
+    .expand-filters {
+      button {
+        text-align: left;
+        color: #39acaa;
```
Comment (neotek) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/components/aggregation/aggregation.component.scss]: Add to theme.

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/styles/aggregation.component.scss [hunk 11]
```diff
@@ -24,6 +35,11 @@ pp-aggregation-item {
             width: calc(100% + 35px) !important;
             height: 2px !important;
         }
+        &.expanded {
```
Comment (neotek) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/styles/aggregation.component.scss]: Remove

File: src/picturepark-sdk-v1-angular/src/app/content-picker/components/contents-picker/contents-picker.component.html [hunk 12]
```diff
@@ -9,9 +9,12 @@
       (previewItemChange)="openDetails($event)"
       (selectedItemsChange)="selectionChange($event)">
     </pp-content-browser>
+    <div class="expand-browser" *ngIf="expandedContent" (click)="expand()">
+      <i class="material-icons">keyboard_arrow_left</i>
+    </div>
   </div>
 
-  <div class="filters">
+  <div class="filters" #filtersContainer>
```
Comment (neotek) [inline src/picturepark-sdk-v1-angular/src/app/content-picker/components/contents-picker/contents-picker.component.html]: remove

File: src/picturepark-sdk-v1-angular/src/app/content-picker/components/contents-picker/contents-picker.component.ts [hunk 13]
```diff
@@ -1,11 +1,11 @@
-import { Component, OnInit, OnDestroy, ViewChild, ElementRef } from '@angular/core';
+import { Component, OnInit, OnDestroy, ViewChild, OnChanges, SimpleChanges, Input } from '@angular/core';
```
Comment (neotek) [inline src/picturepark-sdk-v1-angular/src/app/content-picker/components/contents-picker/contents-picker.component.ts]: Revert

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/components/aggregation-list/aggregation-list.component.scss (hunk 9 above)
Comment (tapmapp) [reply src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/components/aggregation-list/aggregation-list.component.scss]: we made a mistake on this. must stay on the ui lib

**Pull request merged**